crates/sui-framework/packages/dwallet-system/sources/dwallet.move

```diff
@@ -451,17 +453,50 @@ module dwallet_system::dwallet {
 
     /// Register an encryption key to encrypt a user share.
     /// The key is saved as an immutable object.
-    public fun register_encryption_key(key: vector<u8>, scheme: u8, ctx: &mut TxContext): ID {
+    public fun register_encryption_key(key: vector<u8>, scheme: u8, ctx: &mut TxContext) {
         assert!(is_valid_encryption_key_scheme(scheme), EInvalidEncryptionKeyScheme);
         let encryption_key = EncryptionKey {
             id: object::new(ctx),
             scheme,
             encryption_key: key,
             key_owner_address: tx_context::sender(ctx),
         };
-        let encryption_key_id = object::id(&encryption_key);
         transfer::freeze_object(encryption_key);
-        encryption_key_id
+    }
+
+    struct ActiveEncryptionKeys has key {
+        id: UID,
+        encryption_keys: Table<address, ID>,
+    }
+
+    public fun create_active_encryption_keys(ctx: &mut TxContext) {
```

Comment:
I think this function should be called once during initialization of this module

Reply:
This is correct and aligns with the initial idea, but Omer mentioned that it's too complex to implement in Move at this stage and should be addressed separately in the future.

Reply:
Please open an issue for this

Reply:
https://github.com/dwallet-labs/dwallet-network/issues/157